app/assets/stylesheets/application/_custom/allegheny.scss

```diff
@@ -11,11 +11,11 @@ $brand-warning-l: #FBE28D;
 
 $brand-danger: #CD0D0D;
 $brand-danger-d: #680A1D;
-$brand-danger-l: #AD7E88;
+$brand-danger-l: #E1B7C4;
```

Comment:
@awisema responding here so we have a thread. I think you are referencing the screenshot which is a change from 
<img width="741" alt="Screen Shot 2022-07-05 at 3 58 26 PM" src="https://user-images.githubusercontent.com/5333982/177345730-f4b7af85-72db-42b1-9c30-06562b4d1571.png">
to
<img width="750" alt="Screen Shot 2022-07-05 at 3 58 53 PM" src="https://user-images.githubusercontent.com/5333982/177345751-65e15064-259a-44eb-beca-23e67bb6d6ac.png">


There are certainly brighter colors we could use. What I did was lighten the existing colors until there was enough contrast. I'm not sure the history behind we why have certain colors for certain clients –– maybe they didn't want it to look too alarming. For some other clients, we use these colors for `brand-danger-l`:

<img width="745" alt="Screen Shot 2022-07-05 at 4 00 34 PM" src="https://user-images.githubusercontent.com/5333982/177345918-886e6bb3-9c25-47c9-b3f4-f4090bc75c6f.png">


<img width="754" alt="Screen Shot 2022-07-05 at 4 00 44 PM" src="https://user-images.githubusercontent.com/5333982/177345911-c3a1c6b6-8cd4-4b96-8508-f0167a862ec8.png">

Reply:
I agree with Sandy, we need to make the danger more red, less pink.  Otherwise, this looks great!

Reply:
Ok, updated the "rose" danger to the following. You can see which themes changed in https://github.com/greenriver/hmis-warehouse/pull/2302/commits/a118ec6e9faae634a3ccd8a570c66d372ac3c6f6

<img width="754" alt="Screen Shot 2022-07-06 at 1 03 07 PM" src="https://user-images.githubusercontent.com/5333982/177536325-0aee6717-652c-43a4-9c86-2a4e26048098.png">

Reply:
I think I'm ok with that.  I think the color I'm looking for would need white text.  Here's a screenshot of an upcoming design that leans on the success and danger colors.  The lighter red is probably fine, but definitely not as in-your-face bright.    Maybe as a second pass we should provide all three (Success, Warning, Danger) with dark versions where white text would be the appropriate contrasting color. 

(These also aren't as dark as I was remembering)
<img width="1052" alt="Screen Shot 2022-07-06 at 7 25 03 AM" src="https://user-images.githubusercontent.com/1346876/177539881-76c51c00-66a0-4533-9e91-19acb78bc813.png">

Reply:
Yep so the screenshot you shared is a `#FFA69B` background, which is the same as my last screenshot in this comment https://github.com/greenriver/hmis-warehouse/pull/2302#discussion_r913837738. I agree it could be lighter, even though it does pass AAA with black text as-is. This color definitely can't have white text on it. I'll lighten in a bit.

This is how it works with the brand colors:
* `$brand-danger` is used for red crosses, or as red text on white background.
* `$brand-danger-d` is used as a background with white text on top
* `$brand-danger-l` is used as a background with black text on top

What I did in this PR is go through all of those and make sure we have sufficient contrast for each theme to be used in the above ways.

Reply:
@eanders I lightened the `$brand-success-l` values a bit more too. Here is the default, you can compare to https://qa-warehouse.openpath.host/style_guide/alerts
<img width="863" alt="Screen Shot 2022-07-06 at 1 49 13 PM" src="https://user-images.githubusercontent.com/5333982/177543658-90984bc1-8483-4587-98a1-8e36d1ebc18f.png">